suite/runsuite.cmake

```diff
@@ -319,30 +319,28 @@ if (UNIX AND ARCH_IS_X86)
   set(ENV{CXXFLAGS} "")
   set(prev_run_tests ${run_tests})
   set(run_tests OFF) # build tests but don't run them
-  if (NOT cross_android_only)
```

Comment:
yep, so now the behavior for both cross-only options is the same, i.e. adding the configurations, but only failing if the respective variable is set.